docs/src/main/sphinx/connector/hive.rst

```diff
@@ -622,6 +622,23 @@ configuration file ``hive-site.xml`` and restart the metastore service:
         <value>org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader</value>
     </property>
 
+File-based metastore
+^^^^^^^^^^^^^^^^^^^^
+
+In order to use a file-based metastore, you must configure the metastore with:
+
+.. code-block:: text
+
+    hive.metastore=file
+    hive.metastore.catalog.dir={S3 or HDFS file}
+
+.. warning::
+
+  File-based metastore is an experimental feature used for development. It
+  should not be used as a replacement for Thrift or Glue metastores. You
+  should not rely on any of the metadata files generated by the file-based
+  metastore. It should only be used to export data.
```

Comment:
it should be used for nothing. You can easily have a situation where a file based metastore from older version is no longer readable on newer version.

documenting this will increase the number of questions we get - not reduce.